File: Data Structures & Algorithms/is-anagram/submission-2.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        res=[]
        x=list(s)
        y=list(t)
        for i in x:
            if len(x)>=len(y):
                if i in y:
                    res.append(i)
                    y.remove(i)
        a="".join(map(str,res))
        if a==s:
            return True
        else:
            return False

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        //List<Integer> l=new ArrayList<>();
        int[] res=new int[nums.length];
        for(int i=0;i<nums.length;i++){
            int prod=1;
            for(int j=0;j<nums.length;j++){
                if(i==j){
                    continue;
                }
                else{
                    prod*=nums[j];
                }
            }
            res[i]=prod;
        }
        return res;
    }
}  
